src/cloud-api-adaptor/test/e2e/main_test.go

```diff
@@ -173,6 +173,17 @@ func TestMain(m *testing.M) {
 			}
 		}
 
+		// Check if provisioner supports PodVM instance creation and VM_POOL_IPS is empty
+		if vmHandler, ok := provisioner.(pv.PodVMInstanceHandler); ok {
+			currentProps := provisioner.GetProperties(ctx, cfg)
+			if currentProps["VM_POOL_IPS"] == "" {
```

Comment:
I appreciate that the e2e test is somewhat e2e, but documentation of this would be great as I had issues where the CAA deploy was timing out everytime (a separate issue that I use `TEST_INSTALL_CAA=no` to resolve), but that meant it kept trying to create VMs each time until I found this way to skip it, so this feature was really useful and let me get the test passing!

Reply:
Sure. I'll document this in the e2e README in the next iteration.